src/NuGet.Indexing/Extraction/DocumentCreator.cs

```diff
@@ -26,28 +27,36 @@ public static IndexWriter CreateIndexWriter(Lucene.Net.Store.Directory directory
             return indexWriter;
         }
 
-        public static IDictionary<string, string> CreateCommitMetadata(string description, string trace)
+        public static IDictionary<string, string> CreateCommitMetadata(DateTime commitTimeStamp, string description, string trace)
         {
             IDictionary<string, string> commitMetadata = new Dictionary<string, string>();
-            commitMetadata.Add("commitTimeStamp", DateTime.UtcNow.ToString("O"));
+            commitMetadata.Add("commitTimeStamp", commitTimeStamp.ToString("O"));
+            commitMetadata.Add("commit-time-stamp", commitTimeStamp.ToString("O"));
```

Comment:
I've done a lot of investigation right now and it seems like we have [some diagnostic/monitoring](https://github.com/search?l=&q=commit-time-stamp+user%3Anuget+language%3AC%23&ref=advsearch&type=Code&utf8=%E2%9C%93) tools that depend on this.